.github/sync-repo-settings.yaml

```diff
@@ -0,0 +1,21 @@
+rebaseMergeAllowed: true
+squashMergeAllowed: true
+mergeCommitAllowed: false
+branchProtectionRules:
+- pattern: main
+  isAdminEnforced: true
+  requiredStatusCheckContexts:
+    - gcb-presubmit-node14 (firestore-nodejs-getting-start)
+    - cla/google
+    - Auto-approve.yml check
```

Comment:
This doesn't need to be required, but it should be a check

Reply:
removed